lib/src/front_end/ast_node_visitor.dart

```diff
@@ -512,48 +521,53 @@ class AstNodeVisitor extends ThrowingAstVisitor<Piece> with PieceFactory {
     if (node.members.isEmpty) {
       // If there are no members, format the constants like a delimited list.
       // This keeps the enum declaration on one line if it fits.
+      // TODO(tall): The old style preserves blank lines and newlines between
+      // enum values. A newline will also force the enum to split even if it
+      // would otherwise fit. Do we want to do that with the new style too?
```

Comment:
Hmm, I think I like having the ability to split my enum values?, even if it would fit on one line. It seems more clear to me.

Reply:
> Hmm, I think I like having the ability to split my enum values?

Me too. I think I didn't add support for that to the new formatter because there wasn't a clear mechanism in place to implement it (currently, we really only force newlines and blank lines around comments), but I think we probably want it here. That's what the TODO is for. :)